include/assert.hpp

```diff
@@ -33,8 +33,11 @@
 
 #ifdef ASSERT_USE_MAGIC_ENUM
  // this is a temporary hack to make testing thing in compiler explorer quicker (it disallows simple relative includes)
- #include \
- "../third_party/magic_enum.hpp"
+ #if __has_include("magic_enum/magic_enum.hpp")
```

Comment:
Is the issue here that the header has a different path depending on FetchContent or find_package? This can work but I'm wondering if there might be a better way to go about this.

Reply:
magic_enum 0.9.4 change to magic_enum/magic_enum.hpp that's why (see https://github.com/Neargye/magic_enum/releases/tag/v0.9.4)

Reply:
Ahh, thanks for the explanation!